src/api/Nhs.Appointments.Api/Validators/GetAvailabilityCreatedEventsRequestValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using FluentValidation;
+using Nhs.Appointments.Api.Models;
+
+namespace Nhs.Appointments.Api.Validators;
+
+public class GetAvailabilityCreatedEventsRequestValidator : AbstractValidator<GetAvailabilityCreatedEventsRequest>
+{
+    public GetAvailabilityCreatedEventsRequestValidator()
+    {
+        RuleFor(x => x.Site)
+            .NotEmpty()
+            .WithMessage("Provide a valid site");
+
+        RuleFor(x => x.From)
```

Comment:
You don't need this validation here if you make the From property a DateOnly the new validator will take care of it